### ISSUE TITLE
getting error while creating wallet 

### ISSUE THREAD

**Issue by hi0001234d** [CityOfZion/neo-python#370]
### Current behavior

Getting the error "[E 180403 10:45:13 UserWallet:94] Could not build database unable to open database file" while running command **create wallet mywallet4**

I think its because blockchain np-prompt client was closed unexpectedly last time as I'm using putty to run it so let me know if that's the issue. And how to fix it, I think if I keep running np-prompt forever on test net than its solved but I've no idea how to do that

### How to reproduce

It happens with latest neo-python clone and CentOS 7.4.1708 right during setup of neo-python

### Your environment

- OS: CentOS 7.4.1708
- neo-python version: latest clone from Github np-prompt command not working yet.
- Python version: 3.6

**Comment by dethos** [comment]
@hi0001234d I didn't try to reproduce the issue yet, but as far as I know, the latest versions of `neo-python` require python 3.6. 

Even if that is not the cause of this particular problem, it will be of the next ones.

**Comment by hi0001234d** [comment]
Sorry its python 3.6. Its my mistake to specify 3.5 here from my last thread actually. 

Let me know if you know the cause.

**Comment by hi0001234d** [comment]
Solved! closing it :-)

It was my silly mistake that I was giving the wrong wallet file path, it happened because of I had moved some of the wallet directories here and there. 

Anyway error was not so intuitive so to find out the solution I deleted the client twice and installed block chain from scratch and even after that when it not fixed had went to the actual script throwing error did some more error logging and finally found out that its my file path that is wrong. fixed and worked! 

Hope it will help some one in future.
